### PR TITLE
[CSM-417] Bump `daedalus-bridge` to `psc 0.11.x`

### PR DESCRIPTION
**Pull request by sectore**
How to test?
```
$ cardano-sl » scripts/build/cardano-sl.sh
$ cardano-sl » stack exec -- cardano-wallet-hs2purs

$ cardano-sl » tmux
scripts/launch/demo-with-wallet-api.sh
```

```
$ cardano-sl » cd daedalus
$ cardano-sl/daedalus » rm -rf output node_modules bower_components
$ cardano-sl/daedalus » yarn
$ cardano-sl/daedalus » yarn build:prod
```

I've tested following endpoints:
```
$ cardano-sl/daedalus » node
> var api = require('../output/Daedalus.ClientApi')
undefined
> var js = require('../src/tls_workaround.js')
undefined
> var ca = js.readCA('/XXX/cardano-sl/scripts/tls-files/ca.crt')
undefined
> var tls = api.tlsInit(ca)
undefined
> api.syncProgress(tls).then(console.log).catch(console.log)
Promise { <pending> }
> { _spPeers: 0,
  _spNetworkCD: { getChainDifficulty: { getBlockCount: 61 } },
  _spLocalCD: { getChainDifficulty: { getBlockCount: 61 } } }
> api.getWallets(tls).then(console.log).catch(console.log)
Promise { <pending> }
> []
api.importWallet(tls, '/XXX/cardano-sl/keys/2.key', null).then(console.log).catch(console.log)
Promise { <pending> }
> { cwPassphraseLU: 1503314418.7698858,
  cwMeta: { cwUnit: 0, cwName: 'Genesis wallet', cwAssurance: 'CWANormal' },
  cwId: 'LaVWX1Lfkqgzb6sDLnUvyxUYunUvBmYspQH4cnLV2UtYartph9kL',
  cwHasPassphrase: false,
  cwAmount: { getCCoin: '100' },
  cwAccountsNumber: 1 }
> api.importWallet(tls, '/XXX/cardano-sl/keys/3.key', null).then(console.log).catch(console.log)
Promise { <pending> }
> { cwPassphraseLU: 1503314425.3558917,
  cwMeta: { cwUnit: 0, cwName: 'Genesis wallet', cwAssurance: 'CWANormal' },
  cwId: 'LaVWPVaMHxNVtqJ1uvVZ8FyQmeRam5avHE1Uv9iwRivCKTN83CUW',
  cwHasPassphrase: false,
  cwAmount: { getCCoin: '0' },
  cwAccountsNumber: 1 }
> api.getWallets(tls).then(console.log).catch(console.log)
Promise { <pending> }
> [ { cwPassphraseLU: 1503314418.7698858,
    cwMeta: { cwUnit: 0, cwName: 'Genesis wallet', cwAssurance: 'CWANormal' },
    cwId: 'LaVWX1Lfkqgzb6sDLnUvyxUYunUvBmYspQH4cnLV2UtYartph9kL',
    cwHasPassphrase: false,
    cwAmount: { getCCoin: '100' },
    cwAccountsNumber: 1 },
  { cwPassphraseLU: 1503314425.3558917,
    cwMeta: { cwUnit: 0, cwName: 'Genesis wallet', cwAssurance: 'CWANormal' },
    cwId: 'LaVWPVaMHxNVtqJ1uvVZ8FyQmeRam5avHE1Uv9iwRivCKTN83CUW',
    cwHasPassphrase: false,
    cwAmount: { getCCoin: '0' },
    cwAccountsNumber: 1 } ]
```

### CODE REVIEW

File: daedalus/package.json [hunk 1]
```diff
@@ -23,34 +23,34 @@
...
-    "sinon": "^2.1.0",
+    "sinon": "^3.2.1",
     "text-encoding": "^0.6.4",
-    "webpack": "2.3.3",
-    "webpack-dev-server": "2.4.2",
+    "webpack": "3.5.5",
+    "webpack-dev-server": "2.7.1",
     "xhr2": "^0.1.4"
   },
   "//": "NOTE: if you will be bumping bip39 to >=2.2.0 be aware of https://issues.serokell.io/issue/VD-95 . In this case you will have to modify how we validate paperVendMnemonics. Affected endpoints: isValidMnemonic, redeemAdaPaperVend",
```
Comment (akegalj) [inline daedalus/package.json]: Note this warning. I will rollback bip39 to 2.2.0

**Comment by domenkozar** [comment]
@sectore something went wrong and wrong appveyor CI builds to point @neongreen account, can you push another dummy commit to retrigger CI?

**Comment by akegalj** [comment]
@sectore @domenkozar has this been addressed ?
> something went wrong and wrong appveyor CI builds to point

**Comment by domenkozar** [comment]
Looks good!

**Pull request merged**